### ISSUE TITLE
mode 2 not stopping at wait point

### ISSUE THREAD

**Issue by nbhubbard96** [Courseplay/courseplay#5732]
im using a grain cart (i know its not supported fully but it works pretty well) to unload a combine. when the cart gets full it goes and runs the course like it should but it doesn't stop at the waiting point no matter how long i set the time for. im using the in game elmers cart

**Comment by lxfTTjek** [comment]
None yet

**Comment by lxfTTjek** [comment]
None yet

**Comment by FarmerMoe** [comment]
In the attached file, are are some minor changes to allow chaser wagons to unload a combine.  Create the normal run course as you would for a wagon but put an unload point where you want the chaser wagon to unload -- keep in mind that you must manually account for the pipe offset when recording the course.

Within the file, in the comments are 3 changes that need to be made to courseplay.lua and AIDriver.lua. Finally, the file needs to be renamed to EBP.lua and added to the courseplay zip file along with the modified courseplay.lua and AIDriver.lua files.

[EBP.lua.txt](https://github.com/Courseplay/courseplay/files/4940271/EBP.lua.txt)

**Comment by nbhubbard96** [comment]
i tried to do this but when i loaded the game up the save game got stuck in loading and never would load fully

**Comment by FarmerMoe** [comment]
Keep in mind these changes are in no way part of the official code or release plans for Courseplay and no support is provided by the Courseplay team.

That said, without looking at the actual changes that you made or seeing the error logs I suspect that you have not edited the files correctly. ... I made this mistake multiple times as I figured out the lua coding syntax. In my experience I found that this problem would occur when I made a simple syntax error. I would suggest checking the file syntax by using an lua editor such as lua edit 2010. 

Assuming that you are using Courseplay for FS19 v6.02.00040 the following files in the attached zip should be added to the courseplay.zip file for the changes to work.
[GrainWagon changes.zip](https://github.com/Courseplay/courseplay/files/4943190/GrainWagon.changes.zip)


@courseplay team: Keep up the great work! Looking forward to future releases and your fully functional Mode 3.
